sorting/selection-sort.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Implement [Selection sort]
+ *
+ * Copyright (c) 2018, Uliana a.k.a. "The Nullable"
+ * Licensed under the MIT License.
+ */
+
+'use strict';
+
+/**
+ * Selection sort algorithm.
+ * Complexity: O(N^2).
+ *
+ * @example
+ * console.log(selectionSort([2, 5, 1, 0, 4])); // [ 0, 1, 2, 4, 5 ]
+ *
+ * @public
+ * @module sorting/bubblesort
+ * @param {Array} array Input array.
+ * @return {Array} Sorted array.
+ */
+function selectionSort(array) {
+  var minIdx;
```

Comment:
I think we should use `let` to declare variable than using `var` keyword

Reply:
Fixed!